.github/workflows/test.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       - name: Set up elixir
         uses: erlef/setup-beam@v1
         with:
-          otp-version: 21.0
-          elixir-version: 1.8
+          otp-version: 24.0
+          elixir-version: 1.12
```

Comment:
any reason not to go to 1.13?

Reply:
Other projects that use this library run on 1.12